Feature/agnieszka/products desktop

AgnieszkaBaranowska opened the pull request:
wersja na desktop+ dodanie hovera, 
Te 3 kropki w kartach produktów zrobiłam w html zamiast pseudoelement

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
UPDATE LOCAL HOST

DorotaDomanska committed:
Hero desktop

DorotaDomanska committed:
Hero desktop

AgnieszkaBaranowska committed:
products desktop css

AgnieszkaBaranowska committed:
products desktop css

AgnieszkaBaranowska committed:
products desktop css

AgnieszkaBaranowska committed:
update products html+css

AgnieszkaBaranowska committed:
update products html+css

AgnieszkaBaranowska committed:
update products html+css

kwiatkowsko reviewed (changes requested):
Dodaj proszę na widoku tabletu większy odstęp do dołu od buttona.

KitsuneAkvma committed:
Update package.json

aktualizacja ścieżki

KitsuneAkvma committed:
Update package.json

zmaina ścieżki

AgnieszkaBaranowska committed:
update products css

AgnieszkaBaranowska committed:
update products css

AgnieszkaBaranowska committed:
update products css

AgnieszkaBaranowska commented:
poprawione :)

MalgorzataMarczynska committed:
poprawka buga ze zdjęciem krowy

KitsuneAkvma committed:
reviews slide structure update

DorotaDomanska committed:
Drobne poprawki do hero desktop

DorotaDomanska committed:
Drobne poprawki do hero desktop

AgnieszkaBaranowska committed:
Merge branch 'feature/Agnieszka/products-desktop' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/Agnieszka/products-desktop

kwiatkowsko committed:
add hover and transition

kwiatkowsko committed:
add hover and transition

AgnieszkaBaranowska committed:
products - animation, link

AgnieszkaBaranowska committed:
products - animation, link

kwiatkowsko committed:
Update _setup.scss

kwiatkowsko committed:
Update _setup.scss

kwiatkowsko committed:
Merge pull request #65 from KitsuneAkvma/bugfix/cow-image

fix image-cow-desktop

KitsuneAkvma committed:
Okiełznano stukturę, style i  chowanie reszty elementów

kwiatkowsko committed:
Merge branch 'feature/DorotaDomanska/hero__desktop' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/DorotaDomanska/hero__desktop

kwiatkowsko committed:
Update hero.html

kwiatkowsko committed:
Merge pull request #60 from KitsuneAkvma/feature/DorotaDomanska/hero__desktop

Hero desktop

kwiatkowsko committed:
Update _contacts.scss

kwiatkowsko committed:
Merge branch 'feature/Agata/hover-and-shadow' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/Agata/hover-and-shadow

KitsuneAkvma committed:
Merge pull request #66 from KitsuneAkvma/feature/Agata/hover-and-shadow

Agata/Contacts/desktop + hover patch

kwiatkowsko commented:
Ostatnie poprawki! Obiecuję! 

- [ ] transform: scale(1.02); - dla komórki 

- [ ] transform: scale(1.03); - dla tabletu

- [ ] wyrzucenie hovera z buttona
Ślicznie będzie!

KitsuneAkvma committed:
Działający slider

Kod jest toporny ale udało mi się to skleić. Niestety z moją mierną znajomością JS tyle udało mi się zrobić

KitsuneAkvma committed:
Merge pull request #68 from KitsuneAkvma/Reviews/slider

Reviews/slider added, functioning, no conflict

KitsuneAkvma committed:
Imiona i lokalizacje, aktualizują się  teraz

KitsuneAkvma committed:
Merge pull request #69 from KitsuneAkvma/Reviews/slider

Imiona i lokalizacje teraz się aktualizują

AgnieszkaBaranowska committed:
poprawki

AgnieszkaBaranowska committed:
update products

AgnieszkaBaranowska committed:
Merge branch 'feature/Agnieszka/products-desktop' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/Agnieszka/products-desktop

kwiatkowsko reviewed (changes requested):
![image](https://user-images.githubusercontent.com/96976740/188972693-afb39aa9-847e-453d-94af-5fc4d593e8b2.png)
ostatnia zmiana - kropki się zrobiły podwójne
rebase i pójdzie :)
Świetna robota!

kwiatkowsko commented:
+ czy można usunąć te 200 linii zakomentowanego kodu? :D

kwiatkowsko commented:
zmiany dodane w innym branchu
